mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import time
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = {}
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": int(time.time() * 100),
```

Comment:
(In other words, you need to do `session_store.clock.seconds()` and not `time.time()`)